package.json

```diff
@@ -55,15 +55,17 @@
     "url": "https://github.com/wildbit/postmark.js/issues"
   },
   "precommit": [
-    "test"
+    "compile",
+    "test",
+    "compile-docs"
   ],
   "devDependencies": {
     "@types/chai": "^4.1.4",
     "@types/mocha": "^5.2.5",
     "@types/request": "^2.47.1",
     "chai": "^4.1.2",
     "merge": "1.2.0",
-    "mocha": "3.5.3",
```

Comment:
yeah, I left that one for old library version, but.. we are moving to new one, so no point in leaving it, it was super old.